### PR TITLE
chore: update wearables AB Converter with new endpoint

### PR DESCRIPTION
**Pull request by pravusjif**
* Updated wearables AB Converter to fetch from the new endpoint (previous endpoint will be deprecated soon)
* Created new structure to parse new endpoint JSON response (we can no longer use WearableItem class directly as the JSON structure has changed)
* Updated wearables AB Converter logic to fetch the different "pages" of wearables (the new endpoint doesn't retrieve all the wearables like the previous one)

### CODE REVIEW

**Review by pbosio — APPROVED** [review]

File: unity-client/Assets/ABConverter/Client.cs [hunk 1]
```diff
@@ -61,18 +61,128 @@ public Settings(ContentServerUtils.ApiTLD tld = ContentServerUtils.ApiTLD.ORG)
...
+                
+                foreach (var wearableData in deployments)
+                {
+                    // Populate new WearableItem with fetched data
+                    WearableItem wearable = new WearableItem()
+                    {
+                        id = wearableData.metadata.id,
+                        category = wearableData.metadata.data.category,
+                        tags = wearableData.metadata.data.tags,
+                        baseUrl = "https://peer.decentraland.org/content/contents/",
```
Comment (D4rWiNSS) [inline unity-client/Assets/ABConverter/Client.cs]: Can we move this to a constant?

File: unity-client/Assets/ABConverter/Client.cs [hunk 2]
```diff
@@ -370,7 +480,7 @@ public static void DumpAllNonBodiesWearables()
             EnsureEnvironment();
 
             // For debugging purposes we can intercept this item list with LinQ for specific wearables
-            List<WearableItem> avatarItemList = GetAvatarMappingList("https://wearable-api.decentraland.org/v2/collections")
+            List<WearableItem> avatarItemList = GetAvatarMappingList("https://peer.decentraland.org/content/deployments?entityType=wearable&onlyCurrentlyPointed=true")
```
Comment (D4rWiNSS) [inline unity-client/Assets/ABConverter/Client.cs]: can we move this to constants?

**Review by D4rWiNSS — APPROVED** [review]
LGTM! Just small suggestions

File: unity-client/Assets/ABConverter/Client.cs (hunk 1 above)
Comment (pravusjif) [reply unity-client/Assets/ABConverter/Client.cs]: good idea, done.

File: unity-client/Assets/ABConverter/Client.cs (hunk 2 above)
Comment (pravusjif) [reply unity-client/Assets/ABConverter/Client.cs]: good idea, done.

**Review by sandrade-dcl — APPROVED** [review]

**Pull request merged**